tests/utils.go

```diff
@@ -1775,11 +1758,11 @@ func ExecuteCommandOnPodV2(virtCli kubecli.KubevirtClient, pod *k8sv1.Pod, conta
 		Stderr: &stderrBuf,
 		Tty:    false,
 	}
-	err = execCommandOnPod(virtCli, pod, containerName, command, options)
+	err = ExecCommandOnPod(virtCli, pod, containerName, command, options)
 	return stdoutBuf.String(), stderrBuf.String(), err
 }
 
-func execCommandOnPod(virtCli kubecli.KubevirtClient, pod *k8sv1.Pod, containerName string, command []string, options remotecommand.StreamOptions) error {
+func ExecCommandOnPod(virtCli kubecli.KubevirtClient, pod *k8sv1.Pod, containerName string, command []string, options remotecommand.StreamOptions) error {
```

Comment:
I don't like that we have now `ExecCommandOnPod` and also ExecuteCommandOnPod. Any suggestion on how to improve it is welcome!

Reply:
I share your opinion. The point of this PR is to simplify test/utils API, not to make it more confusing. Maybe you can move both function to a new package (`tests/exec`), and rename them `exec.CommandOnPod` and `exec.CommandOnPodOptions` ? I hope other come up with better names...

Reply:
This is a good option, but it may need a lot of changes since there are too many references to these functions. From a quick search I see:

- `ExecuteCommandOnPod` used in 28 scripts
- `ExecuteCommandOnPodV2` used in 12 scripts

And here `ExecCommandOnPod` will be used only in 1 script.

Reply:
I agree that moving all this stuff to a new pkg. You even got naming suggestions. 

I don't really mind if all goes in one PR, or you prefer to have multiple small ones. I would go big or go home and be done with it (one PR).

Reply:
Alright, let's move it to a new package. Do you prefer to have it in this PR or in a new one?

Reply:
I prefer to reuse this one.

Reply:
Thank you all. Please let's resolve this conversation.